tests/integration/scripteditor.ts

```diff
@@ -127,6 +146,42 @@ describe('Script Editor tests', () => {
     cy.closeTab(-1);
   });
 
+  // test to check if output kernel has expected output
+  it('checks for valid output', () => {
+    openFile('py');
+    cy.get('button[title="Run"]').click({ force: true });
+    cy.get('.elyra-ScriptEditor-OutputArea-output').should(
+      'have.text',
+      'Hello Elyra\n'
+    );
+
+    //close console tab
+    cy.closeTab(-1);
+
+    // Close editor tab
+    cy.closeTab(-1);
+  });
+
+  // test to check if output kernel has Error message for invalid code
+  it('checks for Error message', () => {
+    openFile('py');
+    cy.get('span[role="presentation"]').type('print("test")\n');
```

Comment:
This won't produce an error as we expect for this test, maybe something like `cy.get('span[role="presentation"]').type('print"test"\n')` should do it.